src/python/turicreate/_cython/cy_sframe.pyx

```diff
@@ -53,6 +57,9 @@ cdef pydict_from_gl_error_map(gl_error_map& d):
         inc(it)
     return ret
 
+
```

Comment:
I think the PEP8 police would say you only want two empty lines here 😄

Reply:
Okay